rust/stracciatella_c_api/src/c/librarydb.rs

```diff
@@ -106,18 +94,13 @@ pub extern "C" fn LibraryFile_read(
     buffer: *mut u8,
     buffer_length: size_t,
 ) -> bool {
+    forget_rust_error();
     let file = unsafe_mut(file);
     let buffer = unsafe_slice_mut(buffer, buffer_length);
-    match file.read_exact(buffer) {
-        Err(err) => {
-            remember_rust_error(format!("{:?}", err));
-            false
-        }
-        Ok(_) => {
-            forget_rust_error();
-            true
-        }
+    if let Err(err) = file.read_exact(buffer) {
+        remember_rust_error(format!("{:?}", err));
```

Comment:
Might be good to include library file name in the error message.

Reply:
Done